wireless/wapi/src/wireless.c

```diff
@@ -329,6 +330,7 @@ static int wapi_scan_event(FAR struct iw_event *event,
         /* Reset it. */
 
         bzero(temp, sizeof(struct wapi_scan_info_s));
+        temp->encode = 0xffff;
```

Comment:
if we have has_encode flag, why need change the default value of enocde to 0xffff?

Reply:
#define IW_ENCODE_ENABLED    0x0000  /* Encoding enabled */
Avoid confusion with existing definitions.